tests/integration/conditions.py

```diff
@@ -238,3 +238,56 @@ def passes(self):
         if self.negate:
             return not self.is_within()
         return self.is_within()
+
+
+class PathExists(Condition):
```

Comment:
the other Condition classes have a __str__ value defined as well, which is called when the test fails, for use in debugging error failures. (ie if not Condition.passes: print(Condition) ) Consider adding this debug message to these classes as well.

Reply:
`__repr__` can be another useful attribute for debug output and it + the `__str__` can allow verbose and brief output modes.

Reply:
Thanks for catching that I didn't noticed that the `__str__` was used for debugging and error reporting. I have added the __str__ but omitted the __repr__ because I don't think it is currently used. I want to keep it consistent with the other existing conditions.